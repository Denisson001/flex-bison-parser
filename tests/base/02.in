{
    print(((0+0-0*0)));
    a = 1;
    b = a + a;
    c = b + b + b;
    print(((a + b)) * c / 3);
}