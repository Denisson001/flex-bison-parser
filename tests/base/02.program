{
    print(42);
    print("24");

    number = 123;
    print(number);

    string_s = "321";
    print(string_s);

    print(string_s[0:1]);
    print(string_s[2]);
}