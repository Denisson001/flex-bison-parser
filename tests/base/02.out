42
24
123
321
32
1
